packages/framework-provider-local-infrastructure/src/test-helper/local-queries.ts

```diff
@@ -0,0 +1,36 @@
+//import { DynamoDB } from 'aws-sdk'
+
+//const documentClient = new DynamoDB.DocumentClient()
+
+export class LocalQueries {
+  constructor() {}
+
+  public async events(primaryKey: string, latestFirst = true): Promise<Array<unknown>> {
+    // const output: DynamoDB.QueryOutput = await documentClient
+    //   .query({
+    //     TableName: `${this.stackName}-events-store`,
+    //     ConsistentRead: true,
+    //     KeyConditionExpression: 'entityTypeName_entityID_kind = :v',
+    //     ExpressionAttributeValues: { ':v': primaryKey },
+    //     ScanIndexForward: !latestFirst,
+    //   })
+    //   .promise()
+
+    // return output.Items || []
+    return []
+  }
+
+  public async readModels(primaryKey: string, readModelName: string, latestFirst = true): Promise<Array<unknown>> {
+    // const output: DynamoDB.QueryOutput = await documentClient
+    //   .query({
+    //     TableName: `${this.stackName}-${readModelName}`,
+    //     KeyConditionExpression: 'id = :v',
+    //     ExpressionAttributeValues: { ':v': primaryKey },
+    //     ScanIndexForward: !latestFirst,
+    //   })
+    //   .promise()
+
+    // return output.Items || []
+    return []
+  }
+}
```

Comment:
Skeleton of the class LoadQueries. Here we need to access directly to the nedb database and perform the queries. We also need to change the interface of the methods to avoid using concepts like primaryKey, which is AWS-specific and not generic